pilot/pkg/networking/util/util.go

```diff
@@ -212,20 +212,13 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
 	})
 }
 
-// isProxyVersion checks whether the given Proxy version matches the supplied prefix.
-func isProxyVersion(node *model.Proxy, prefix string) bool {
-	ver, found := node.GetProxyVersion()
-	return found && strings.HasPrefix(ver, prefix)
-}
-
-// Is1xProxy checks whether the given Proxy version is 1.x.
-func Is1xProxy(node *model.Proxy) bool {
-	return isProxyVersion(node, "1.")
-}
-
-// Is11Proxy checks whether the given Proxy version is 1.1.
-func Is11Proxy(node *model.Proxy) bool {
-	return isProxyVersion(node, "1.1")
+// IsProxyVersionGE11 checks whether the given Proxy version is greater than or equals 1.1.
+func IsProxyVersionGE11(node *model.Proxy) bool {
+	ver, _ := node.GetProxyVersion()
+	if ver >= "1.1" {
```

Comment:
do strings have arithmetic comparison operators like this?

Reply:
yes, https://golang.org/src/strings/compare.go